CorsixTH/Lua/cheats.lua

```diff
@@ -83,11 +83,12 @@ function Cheats:cheatResearch()
 end
 
 function Cheats:cheatEmergency()
-  local outcome, err = self.hospital:createEmergency()
+  local err = self.hospital:createEmergency() -- no err is nil
```

Comment:
It is not customary to denote meaning of return values near a call. The reason is that the comment will become misleading when the function is modified, and you typically don't want to dig around in the code finding all spots that document the return value.

Instead, there is one place for this information, the `--!return` line.

What you can do however, is mention it just below line 91:
``` lua
   -- else 'err == nil', meaning success. This case doesn't need special handling.
```
Technically, you're not documenting the return values of the function with such a comment, you're documenting explicitly that you considered the `err == nil` case in this function, and that you decided to do nothing.